packages/components/src/typography/src/Heading.css

```diff
@@ -1,44 +1,64 @@
 :where(.o-ui-heading) {
     margin: 0;
     font-size: inherit;
-    font-weight: 400;
-    font-variation-settings: "wght" 600;
+    font-weight: 680;
```

Comment:
It is possible to use a token for this value `--hop-font-weight-680`

Reply:
What would you use? Actually the issue is that it does not make sense to set a typographic declarations here. As every heading should have a size.

Reply:
ok for the size but here we're talking about weight